classes-solutions.js

```diff
@@ -0,0 +1,55 @@
+// // 1)
+
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+
+  ageOneYear() {
+    this.age++;
+  }
+
+  eating() {
+    console.log("mmm, mmm, mmm, I'm love'n it");
+  }
+};
+
+let deyvi = new Human('deyvi', 18);
+// console.log(deyvi.age);
+//
+// deyvi.ageOneYear();
+//
+// console.log(deyvi.age);
+//
+// deyvi.eating();
+
+// // 2)
+
```

Comment:
Small note: variable names that are classes should begin with an uppercase.
Great job otherwise!